setup.py

```diff
@@ -69,8 +70,20 @@ def package_files(directory):
     return paths
 
 
-# this sets the __version__ variable
-exec(read_file(path.join("src", "cocotb", "_version.py")))
+__version__ = "2.0.0.dev0"
```

Comment:
While we're here, let's sort out what the proper version string should be.

Reply:
Well we aren't releasing 2.0.0, we are releasing 2.0, so I'd think 2.0.dev0 or 2.dev0 would be more appropriate, the PEP would shed more light.

Reply:
It will be `2.0`. In the past the initial release has always been `MAJOR.MINOR` and patch versions only added after the fact. So let's change that to `2.0.dev0` and we can close that out.

Reply:
Done.